src/components/Avatar/index.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { DefaultProps } from '../../common-types';
+import {
+	Avatar as GrommetAvatar,
+	AvatarProps as GrommetAvatarProps,
+} from 'grommet';
+
+export interface AvatarProps
+	extends DefaultProps,
+		Omit<GrommetAvatarProps, 'size'> {
+	email?: string;
+	firstName?: string;
+	lastName?: string;
+	emphasized?: boolean;
+}
+
+const getInitials = (firstName?: string, lastName?: string) =>
+	`${firstName?.charAt(0).toUpperCase() || ''}${
+		lastName?.charAt(0).toUpperCase() || ''
+	}`;
+
+export const Avatar = ({
+	src,
+	firstName,
+	lastName,
+	emphasized,
+	...otherProps
+}: AvatarProps) => {
+	const [initials, setInitials] = React.useState<string>();
+
+	React.useEffect(() => {
```

Comment:
We don't really need an effect and state here, we can just have 
```
const initials = getInitials(firstName, lastName)
```

You can still calculate the initials, even if `src` is passed